lib/versioning/pep440/index.spec.ts

```diff
@@ -84,71 +84,84 @@ describe('versioning/pep440/index', () => {
+    ${'<2.0.0'}             | ${'replace'}     | ${'1.0.0'}     | ${'1.2.3'}   | ${'<2.0.0'}
+    ${'<2.0.0'}             | ${'pin'}         | ${'1.0.0'}     | ${'1.2.3'}   | ${'==1.2.3'}
+    ${'>0.9.8'}             | ${'bump'}        | ${'1.0.0'}     | ${'1.2.3'}   | ${'>0.9.8'}
+    ${'>0.9.8'}             | ${'replace'}     | ${'1.0.0'}     | ${'1.2.3'}   | ${'>0.9.8'}
+    ${'>0.9.8'}             | ${'pin'}         | ${'1.0.0'}     | ${'1.2.3'}   | ${'==1.2.3'}
+    ${'>2.0.0'}             | ${'bump'}        | ${'1.0.0'}     | ${'1.2.3'}   | ${'>=1.2.3'}
+    ${'>2.0.0'}             | ${'replace'}     | ${'1.0.0'}     | ${'1.2.3'}   | ${'>=1.2.3'}
+    ${'>2.0.0'}             | ${'pin'}         | ${'1.0.0'}     | ${'1.2.3'}   | ${'==1.2.3'}
+    ${'>=2.0.0'}            | ${'bump'}        | ${'1.0.0'}     | ${'1.2.3'}   | ${'>=1.2.3'}
+    ${'>=2.0.0'}            | ${'replace'}     | ${'1.0.0'}     | ${'1.2.3'}   | ${'>=1.2.3'}
+    ${'>=2.0.0'}            | ${'pin'}         | ${'1.0.0'}     | ${'1.2.3'}   | ${'==1.2.3'}
+    ${'~=1.1.0, !=1.1.1'}   | ${'bump'}        | ${'1.0.0'}     | ${'1.2.3'}   | ${'~=1.2.3, !=1.1.1'}
+    ${'~=1.1.0, !=1.1.1'}   | ${'replace'}     | ${'1.0.0'}     | ${'1.2.3'}   | ${'~=1.2.3, !=1.1.1'}
+    ${'~=1.1.0, !=1.1.1'}   | ${'pin'}         | ${'1.0.0'}     | ${'1.2.3'}   | ${'==1.2.3'}
+    ${'~=1.1.0,!=1.1.1'}    | ${'bump'}        | ${'1.0.0'}     | ${'1.2.3'}   | ${'~=1.2.3,!=1.1.1'}
+    ${'~=1.1.0,!=1.1.1'}    | ${'replace'}     | ${'1.0.0'}     | ${'1.2.3'}   | ${'~=1.2.3,!=1.1.1'}
+    ${'~=1.1.0,!=1.1.1'}    | ${'pin'}         | ${'1.0.0'}     | ${'1.2.3'}   | ${'==1.2.3'}
+    ${' '}                  | ${'bump'}        | ${'1.0.0'}     | ${'1.2.3'}   | ${' '}
+    ${' '}                  | ${'replace'}     | ${'1.0.0'}     | ${'1.2.3'}   | ${' '}
+    ${' '}                  | ${'pin'}         | ${'1.0.0'}     | ${'1.2.3'}   | ${'==1.2.3'}
+    ${'invalid'}            | ${'bump'}        | ${'1.0.0'}     | ${'1.2.3'}   | ${null}
+    ${'invalid'}            | ${'replace'}     | ${'1.0.0'}     | ${'1.2.3'}   | ${null}
+    ${'invalid'}            | ${'pin'}         | ${'1.0.0'}     | ${'1.2.3'}   | ${'==1.2.3'}
+    ${'===1.0.3'}           | ${'bump'}        | ${'1.0.0'}     | ${'1.2.3'}   | ${null}
+    ${'===1.0.3'}           | ${'replace'}     | ${'1.0.0'}     | ${'1.2.3'}   | ${null}
+    ${'===1.0.3'}           | ${'pin'}         | ${'1.0.0'}     | ${'1.2.3'}   | ${'==1.2.3'}
+    ${'!=1.2.3'}            | ${'bump'}        | ${'1.0.0'}     | ${'1.2.3'}   | ${null}
+    ${'!=1.2.3'}            | ${'replace'}     | ${'1.0.0'}     | ${'1.2.3'}   | ${null}
+    ${'!=1.2.3'}            | ${'pin'}         | ${'1.0.0'}     | ${'1.2.3'}   | ${'==1.2.3'}
+    ${'~=1.1.0,!=1.1.1'}    | ${'unsupported'} | ${'1.0.0'}     | ${'1.2.3'}   | ${'~=1.2.3,!=1.1.1'}
+    ${'>=19.12.2,<20.13.9'} | ${'replace'}     | ${'19.12.2'}   | ${'21.3.1'}  | ${'>=21.3.1,<22.0.0'}
```

Comment:
Very hard to know how to replace this one, because it spanned major ranges for no discernible reason. The result doesn't make too much sense either.. but that's ok

Reply:
yep, this is a wild one 😄.
guess we can remove this test rather than changing any of the current behavior as it is indeed doesnt portrays any "real world" use case.

Reply:
Sure, feel free to remove it if it made your code complex for any good reason. Otherwise OK to keep the test